bin/fenix-qr-codes.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+/* eslint-disable global-require, no-console */
+
+const fs = require('fs');
+const path = require('path');
+
+const fetch = require('node-fetch');
+const QRCode = require('qrcode');
+
+const rootDir = path.join(__dirname, '..');
+const distDir = path.join(rootDir, 'dist');
```

Comment:
I don't think we should use `dist` because it is "reserved" for the build step. The generated files should live in `src/amo/components/QrCode/img/` probably.

Reply:
See my comment above. Once we have the component that you suggest above, and therefore a location into which to write the files, we can make the job write them there. I will file a follow-up.

I think for now keeping them in `dist` is okay.